README.md

```diff
@@ -111,54 +126,58 @@ With Postgres.app: https://docs.djangoproject.com/en/2.0/ref/contrib/gis/install
 
 ## Run your local dev server
 
-    # Since this is a Procfile based app we'll use Foreman
-    # to manage processes
-    # https://github.com/ddollar/foreman
+    # don't forget to have the heroku CLI installed
+    $ shell> npm install -g heroku
 
-    # Check to see if you have a version of ruby installed
-    # if you do, skip to installing the gem
-    # You may also want to install RVM anyway.
-    $ ruby -v
+    # To run your dev server use heroku, which is compatible with procfiles
+    # (recall that a procfile is basically a list of commands to run)
+    $ heroku local -f $git_root/Procfile.local
 
-    # Install RVM and Ruby
-    # This will install RVM (a Ruby version manager)
-    # and the current stable version of Ruby
-    # https://rvm.io/rvm/install
-    $ \curl -sSL https://get.rvm.io | bash -s stable --ruby
 
-    # Install the foreman Gem
-    $ gem install foreman
+Django will run on http://127.0.0.1:8000/
+The react dev server will run on http://127.0.0.1:5000/ (or the next open port) and proxy requests to Django
 
-    # WORKAROUND: If get an error about openssl installing foreman, here's what happened:
-    # Newer versions of OSX deprecated openSSL, leaving many dependencies broken. You need
-    # to reinstall ruby, but specify exactly where your openSSL libraries are.
-    # read more: https://stackoverflow.com/a/42235933
-    # First find out which version of ruby you are using:
-    $ rvm current
-    # Just use the version number, e.g. use "2.4.1" not "ruby-2.4.1"
 
-    # install openssl
-    $ brew install openssl
 
-    # then reinstall ruby with openssl
-    $ rvm reinstall $current_ruby_version --with-openssl-dir=`brew --prefix openssl`
+## Pull Requests and Heroku Builds.
 
-    # To run your dev server use
-    $ foreman start -f $git_root/Procfile.local
+
+Every time you create a PR, heroku will kick off a build to see if it can create
+a server running the version of your PR. You can see the builds here:
+
+https://dashboard.heroku.com/pipelines/514a9ea9-75d3-4056-81cc-24b8aebd5592
```

Comment:
This reminds me, we need to think about the public vs private repo. Right now this repo is public so anyone could kick off PRs and eat through money on heroku.

Reply:
I think we get unlimited private repos as a non-profit, might have to confirm that, but we may want to move it private and for the public repo later.

Reply:
I think we're ok. LMK if I'm missing anything in my thinking, @ericsandine 
* Anyone who can see that URL has to be able to log in.
* We'd have to approve a PR to kick off a build.